pkg/plugin/trans.go

```diff
@@ -150,26 +134,130 @@ func (r resourceTransitioner[T]) handleReserve() (overbudget bool, verdict strin
 // Any permitted increases are required to be a multiple of factor.
 //
 // A pretty-formatted summary of the outcome is returned as the verdict, for logging.
```

Comment:
It would be easier to understand, if those comments would more clearly reflect the difference between `handleRequested` and `handleReserve`.

Reply:
Fair enough - added some comments in bbfca613b2a4fa8b8d650625b4049296475c89f6, although not sure if there's more I should elaborate on.